src/core/body/body.view.d.ts

```diff
@@ -0,0 +1,34 @@
+import {View} from '../view/view';
+import {ColumnModel} from "../column-type/column.model";
+import {Model} from "../infrastructure/model";
+import {getFactory as labelFactory} from '../services/label';
+import {Table} from "../dom/table";
+
+export declare interface IGetValueFactory{
+	(): any;
+}
+
+export declare interface IValueFactory{
+	(column: ColumnModel, labelFactory: labelFactory): any;
```

Comment:
I'm not sure about this. Is it returning result of value factory? Then it should be ```(row: any): any``` and it has already declared in value.d.ts. I think we should use the same interface.